go.mod

```diff
@@ -34,6 +36,8 @@ require (
 	github.com/lib/pq v1.10.2
 	github.com/luthermonson/go-proxmox v0.2.1
 	github.com/madebywelch/anthropic-go/v2 v2.2.0
+	github.com/muesli/clusters v0.0.0-20200529215643-2700303c1762
+	github.com/muesli/kmeans v0.3.0
```

Comment:
Your go.mod and go.sum needs to be cleaned. For example, we don't use kmeans any more